pkg_building.es.Rmd

```diff
@@ -1,513 +1,511 @@
 
-We recommend you to use the [`codemetar` package](https://github.com/ropensci/codemetar) for creating and updating a JSON [CodeMeta](https://codemeta.github.io/) metadata file for your package via `codemetar::write_codemeta()`. It will automatically include all useful information, including [GitHub topics](#grooming). CodeMeta uses [Schema.org terms](https://schema.org/) so as it gains popularity the JSON metadata of your package might be used by third-party services, maybe even search engines. 
+Te recomendamos que utilices la función [`codemetar` paquete](https://github.com/ropensci/codemetar) para crear y actualizar un JSON [CodeMeta](https://codemeta.github.io/) para tu paquete a través de `codemetar::write_codemeta()`. Incluirá automáticamente toda la información útil, incluyendo [Temas de GitHub](#grooming). CodeMeta utiliza [términos de Schema.org](https://schema.org/) por lo que, a medida que gane popularidad, los metadatos JSON de tu paquete podrían ser utilizados por servicios de terceros, incluso por motores de búsqueda.
 
+## Plataformas
 
-## Platforms
+- Los paquetes deben funcionar en todas las plataformas principales (Windows, macOS, Linux).  Se pueden conceder excepciones a los paquetes que interactúan con funciones específicas del sistema, o envoltorios para utilidades que sólo funcionan en plataformas limitadas, pero los autores deben hacer todo lo posible para la compatibilidad entre plataformas, incluyendo la compilación específica del sistema, o la contenerización de utilidades externas.
 
-* Packages should run on all major platforms (Windows, macOS, Linux).  Exceptions may be granted packages that interact with system-specific functions, or wrappers for utilities that only operate on limited platforms, but authors should make every effort for cross-platform compatibility, including system-specific compilation, or containerization of external utilities.
+## API del paquete
 
+### Nombres de funciones y argumentos
 
-## Package API
-
-### Function and argument naming
-
-* Functions and arguments naming should be chosen to work together to form a common, logical programming API that is easy to read, and auto-complete. 
-
-    * Consider an `object_verb()` naming scheme for functions in your package that take a common data type or interact with a common API. `object` refers to the data/API and `verb` the primary action.  This scheme helps avoid namespace conflicts with packages that may have similar verbs, and makes code readable and easy to auto-complete.  For instance, in **stringi**, functions starting with `stri_` manipulate strings (`stri_join()`, `stri_sort()`, and in **googlesheets** functions starting with `gs_` are calls to the Google Sheets API (`gs_auth()`, `gs_user()`, `gs_download()`).
-
-* For functions that manipulate an object/data and return an object/data of the same type, make the object/data the first argument of the function so as to enhance compatibility with the pipe operator (`%>%`).
```

Comment:
Should I add the native pipe `|>` here? I can make a PR to the English version too.

Reply:
yes please @paocorrales